lib/nodes/composites/Composite.ts

```diff
@@ -35,9 +35,7 @@ export class Composite<State> extends Base<State> {
 
 		let firstChild = 0;
 
-		// Support for latched composites - ignored if not latched
-		// wrapped for clarity - not programmatically necessary
-		if (this.latched) {
+		if (this.shouldLatch(state)) {
```

Comment:
Should we just add a method to reset the .running state?  Need to see how it's used on the other side to see if it would be cleaner.

Reply:
So the way the latching works is that this node calculates which child to start from based on the results of the last run - i'm not sure how else you would reset the state? The other side of this is in the PR:
https://github.com/6RiverSystems/mfp_executive/pull/20

Reply:
Was thinking more of an event (onRun, etc.) that would be overridden for a selector or sequence.  In the end it is the same behavior so I'm not sure it is logically any different (onRun -> reset state vs. a overridden method to determine whether it should be latched).